recipes/spdlog/all/test_package/CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+cmake_minimum_required(VERSION 3.15)
+project(test_package CXX)
 
 find_package(spdlog REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-if(TARGET spdlog::spdlog_header_only)
-    target_link_libraries(${PROJECT_NAME} spdlog::spdlog_header_only)
+if(SPDLOG_HEADER_ONLY)
+    target_link_libraries(${PROJECT_NAME} PUBLIC spdlog::spdlog_header_only)
 else()
-    target_link_libraries(${PROJECT_NAME} spdlog::spdlog)
+    target_link_libraries(${PROJECT_NAME} PUBLIC fmt::fmt spdlog::spdlog)
```

Comment:
Why is fmt required? is it being of the transitive dep that's being set in 2.0?

it's always exposed https://github.com/gabime/spdlog/search?q=include+fmt and a public header include

Reply:
It's a leftover. Thank you for reviewing!